### PR TITLE
Handling m97 agent dependency by checking the new parameter existence…

### PR DESCRIPTION
**Pull request by KrishnaAdityaB**
… in the cmdlet

### CODE REVIEW

File: Tasks/DeployAzureResourceGroup/task.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{
+﻿{
```
Comment (KrishnaAdityaB) [inline Tasks/DeployAzureResourceGroup/task.json]: because of this whitespace change, L0 tests are failing. I am fixing them

**Comment by KrishnaAdityaB** [comment]
L0 tests are green now

**Comment by AshwiniChalla** [comment]
Looks fine. Make sure you checkin this to master also.

**Comment by kmkumaran** [comment]
Looks fine.

**Pull request merged**